tests/jobs/conftest.py

```diff
@@ -32,7 +32,8 @@ def test_started_and_expired_applications(app, public_user_email):
             new_app.status = ApplicationStatus.STARTED
             new_app.updated = datetime.now() - relativedelta(days=200)
             db.session.commit()
-            print(f'Test Inactive App Ref - {new_app.reference_number}', flush=True)
```

Comment:
The flush arg is important to make sure the buffered output is logged out to the console at the time of printing.